docs/content/latest/explore/follower-reads/linux.md

```diff
@@ -34,34 +34,30 @@ showAsideToc: true
 
-YugabyteDB also allows you to specify the maximum staleness of data when reading from tablet followers. If the follower hasn't heard from the leader for  10 seconds (by default), the read request is forwarded to the leader. When there is a long distance between the tablet follower and the tablet leader, you might need to increase the duration. To change the duration for maximum staleness, add the [`yb-tserver` `--max_stale_read_bound_time_ms`](../../../reference/configuration/yb-tserver/#max-stale-read-bound-time-ms) flag and increase the value (default is 10 seconds). For details on how to add this flag when using `yb-ctl`, see [Creating a local cluster with custom flags](../../../admin/yb-ctl/#create a-local-cluster-with-custom-flags).
+YugabyteDB also allows you to specify the maximum staleness of data when reading from tablet followers. If the follower hasn't heard from the leader for  10 seconds (by default), the read request is forwarded to the leader. When there is a long distance between the tablet follower and the tablet leader, you might need to increase the duration. To change the duration for maximum staleness, add the [`yb-tserver` `--max_stale_read_bound_time_ms`](../../../reference/configuration/yb-tserver/#max-stale-read-bound-time-ms) flag and increase the value (default is 10 seconds).
 
 ## 1. Create universe
 
-If you have a previously running local universe, destroy it using the following.
-
-```sh
-$ ./bin/yb-ctl destroy
-```
-
 Start a new local universe with three nodes and a replication factor (RF) of `3`.
 
 ```sh
-$ ./bin/yb-ctl --rf 3 create
+$ ./bin/yugabyted start --base_dir=node-1/ --listen=127.0.0.1
+$ ./bin/yugabyted start --base_dir=node-2/ --listen=127.0.0.2 --join=127.0.0.1
+$ ./bin/yugabyted start --base_dir=node-3/ --listen=127.0.0.3 --join=127.0.0.1
 ```
 
 Add 1 more node.
 
 ```sh
-$ ./bin/yb-ctl add_node
+$ ./bin/yugabyted start --base_dir=node-4/ --listen=127.0.0.4 --join=127.0.0.1
 ```
 
 ## 2. Write some data
 
 Download the [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps) JAR file (`yb-sample-apps.jar`) by running the following command.
 
 ```sh
-$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.1/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar 
+$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.2/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
 ```
 
 By default, the YugabyteDB workload generator runs with strong read consistency, where all data is read from the tablet leader. We are going to populate exactly one key with a `10KB` value into the system. Since the replication factor is `3`, this key will get replicated to only three of the four nodes in the universe.
```

Comment:
Extra space before "By default, "

Reply:
I guess the extra space is just an illusion. However, it seems good in VS Code. It's weird.

![image](https://user-images.githubusercontent.com/13635843/119683247-1f3b0980-be61-11eb-857a-8a6c1b18f8ab.png)